protos/processor.proto

```diff
@@ -50,6 +50,32 @@ message TpRegisterResponse {
     Status status = 1;
 }
 
+// The unregistration request from the transaction processor to the validator/executor
+message TpUnRegisterRequest {
+    // the transaction family name for the logic of the handler
+    string family = 1;
+
+    // The version supported;
+    string version = 2;
+
+    //supported encoding of the handler
+    string encoding = 3;
+
+    // The namesapces that this transaction processor handler expects
```

Comment:
Typo 'namesapces'.

Do we really need to specify the namespaces here?  I wouldn't think so, because they don't uniquely identify the TP in any way.